AzureMonitorAgent/agent.py

```diff
@@ -801,7 +801,13 @@ def handle_mcs_config(public_settings, protected_settings, default_configs):
 
         if (data != ''):
             json_data = json.loads(data)
-            if json_data is not None and "proxy.url" in json_data:
+            BypassProxy = False
+            if json_data is not None and "proxy.bypass" in json_data:
+                bypass = json_data["proxy.bypass"]
+                if bypass == "AMA":
```

Comment:
Can I assume this functionality will be documented other than in release notes?

Reply:
yeah, Arc team will document this